src/open_inwoner/scss/components/Contactmomenten/Contactmomenten.scss

```diff
@@ -1,6 +1,27 @@
 .contactmomenten {
   margin-top: var(--spacing-giant);
 
+  & .grid {
```

Comment:
This should probably become a generic thing, since it's the same as for `.cases`

Reply:
Yes, we will need to refactor this (at some point) in the entire project, because some cards are spread over 4 columns (Onderwerpen) and others over 3 (Cases, Vragen, Profiel cards) and they all need responsiveness. But this is okay for now.